test/org/sagebionetworks/bridge/validators/SubpopulationValidatorTest.java

```diff
@@ -0,0 +1,51 @@
+package org.sagebionetworks.bridge.validators;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import org.sagebionetworks.bridge.exceptions.InvalidEntityException;
+import org.sagebionetworks.bridge.models.studies.Subpopulation;
+
+import com.google.common.collect.Sets;
+
+public class SubpopulationValidatorTest {
```

Comment:
Hmm. I thought I had all these in the CriteriaUtilsTest, which now validates the Criteria values (to share with other entities that implement Criteria), but I didn't do that. I'll add them all there.